lib/src/Base/Func/SymbolicParserExprTk.cxx

```diff
@@ -120,7 +121,7 @@ void SymbolicParserExprTk::initialize() const
   symbol_table.add_function("rint", ExprTk_rint);
   symbol_table.add_function("ln", std::log);
   symbol_table.add_function("lngamma", lgamma);
-  symbol_table.add_function("gamma", tgamma);
```

Comment:
**Note:** Free functions should be viable option, in this case tgamma should work.

Documentation: https://github.com/ArashPartow/exprtk/blob/master/readme.txt#L2347

Example of a free function being used: https://github.com/ArashPartow/exprtk/blob/master/exprtk_simple_example_05.cpp#L64

Reply:
@ArashPartow Many thanks for your comments (and for ExprTk which is really great), I will have a closer look.  I had to replace tgamma because of build failures on Windows, see https://ci.appveyor.com/project/openturns/openturns/build/1.0.1785/job/62kb7qp5jil9b1fe